src/services/QbitProvider.ts

```diff
@@ -103,6 +103,10 @@ export default class QBitProvider implements IProvider {
     return this.axios.get('/app/networkInterfaceAddressList', { params }).then(r => r.data)
   }
 
+  async getGeoDetails(geoDetails: string): Promise<string> {
+    return this.axios.get('/app/resolveGeoDetails', { params: { geoDetails } }).then(r => r.data)
+  }
+
```

Comment:
Never called, not sure why to add that
\+ qbit endpoint doesn't exists

Reply:
All provider code was removed. TBH, I wasn't sure what I was doing at the start of this but I have it figured out now 🙌